frontend/scripts/openapi/generator.ts

```diff
@@ -135,6 +135,19 @@ function tsType(s: ReferenceObject | SchemaObject | undefined): string {
     }
   }
 
+  if ("oneOf" in s) {
+    return (
+      s.oneOf
+        ?.map((obj) => {
+          if ("$ref" in obj) {
+            return obj.$ref.substring(obj.$ref.lastIndexOf("/") + 1);
+          }
+          return obj.type || "unknown";
+        })
+        .join(" | ") ?? "unknown"
+    );
+  }
+
```

Comment:
Is this still needed now?

Reply:
I don't think we have to remove this, it's fine if the generator supports a bit more than we need.